src/box/iproto.cc

```diff
@@ -162,7 +166,8 @@ static int iproto_threads_count;
  * properties, and should be accessible from differnent functions
  * in tx thread.
  */
-static struct evio_service tx_binary;
+static struct evio_service tx_binaries[IPROTO_LISTEN_SOCKET_MAX];
+static unsigned tx_binaries_count;
```

Comment:
We strive to avoid using unsigned type. I'd suggest you to use int32_t, uint32_t or whatever with fixed size